server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -104,19 +114,17 @@
   /**
    * The number of ballots audited.
    */
-  private final Integer my_number_of_ballots_audited;
+  private final Integer my_ballots_audited;
```

Comment:
Actually, I'm not going to cascade it widely because I like the naming I'm already using in the actual county dashboard, and the database.